src/components/App.tsx

```diff
@@ -33,12 +33,25 @@ const ContentSelector:React.FunctionComponent = () => {
   );
 };
 
+const Icon:React.FunctionComponent = () => {
+  return (
+    <img
+      src={"..\\..\\public\\logo_transparent.svg"}
```

Comment:
This link will be incorrect after building, there is no public folder in the build.